### ISSUE TITLE
EBS Encryption has the wrong name - 'kms_key_id' should be kms_key_id

### ISSUE THREAD

**Issue by lee-chrisbell** [npalm/terraform-aws-gitlab-runner#421]
There is a difference in the documentation and the actual code for using the kms_key_id to encrypt the EBS volume - 

```
  dynamic "block_device_mappings" {
    for_each = [var.runner_root_block_device]
    content {
      device_name = lookup(block_device_mappings.value, "device_name", "/dev/xvda")
      ebs {
        delete_on_termination = lookup(block_device_mappings.value, "delete_on_termination", true)
        volume_type           = lookup(block_device_mappings.value, "volume_type", "gp3")
        volume_size           = lookup(block_device_mappings.value, "volume_size", 8)
        encrypted             = lookup(block_device_mappings.value, "encrypted", true)
        iops                  = lookup(block_device_mappings.value, "iops", null)
        kms_key_id            = lookup(block_device_mappings.value, "`kms_key_id`", null) // HERE
      }
    }
  }
```

**Comment by kayman-mk** [comment]
@lee-chrisbell Could you please close this issue? Seems that you fixed it with #422, right?

**Comment by lee-chrisbell** [comment]
Looks like @npalm just did.  Sorry about that!
